### PR TITLE
Rail based Forms

### PR DESCRIPTION
**Pull request by Tahirbhalli**
build in 
> HTML
> ruby on rails
there are different types of forms in there like form_for form tag etc

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<% @user = User.find(params[:id]) %>
```
Comment (uche-inyama) [inline app/views/users/edit.html.erb:1]: Remove this implementation, it should be done in the controller, then reference the instance variable in the view.

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<% if false%>
+<%= form_tag(users_path, method: "POST") do %>
+     <%= label_tag(:username, "Username:") %>
+     <%= text_field_tag(:username) %><br><br>
+     <%= label_tag(:email, "email:") %>
+     <%= email_field_tag(:email) %><br><br>
+     <%= label_tag(:password, "password:") %>
+     <%= password_field_tag(:password) %><br><br>
+     <%= submit_tag("Submit")%>
+<% end %>
```
Comment (uche-inyama) [inline app/views/users/new.html.erb:28]: This form should be commented out.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

Use the link as a reference.

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="submit" value="Submit">
+</form>
+<% end %>
```
Comment (uche-inyama) [inline app/views/users/new.html.erb:43]: This form should also be commented out 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))

Use the links as a reference.

Comment (Spayco) [reply app/views/users/new.html.erb:43]: they are commented, using 
<% if false %>
     code here 
<% end %>
this is the best way i found to comment in an erb file as i saw in many posts on stack overflow.

**Comment by Spayco** [comment]
Just to note again, the old for codes are already commented, as I said in the comment above and in the code itself using 

```
<% if false %>
code here
<% end %>
```
this way we comment out multiple lines with it, it's a _hack_ but it does the job.

**Pull request merged**